CHANGES.rst

```diff
@@ -204,6 +204,9 @@ API Changes
   - Removed deprecated ``Pix2Sky_AZP.check_mu`` and ``Pix2Sky_SZP.check_mu``
     methods. [#6170]
 
+  - Deprecated ``GaussianAbsorption1D`` model, as it can be better represented
+    by subtracting ``Gaussian1D`` from a continuum model. [#6200]
```

Comment:
Isn't that too specific? Technically, your continuum can be any model you want, even a blackbody...